src/utils/timing.js

```diff
@@ -2,27 +2,32 @@
 import type { TimingItemType } from '../types';
 
 const timingMap: { [string]: number } = {};
+const preciseTimingMap: { [string]: number } = {};
 const log: TimingItemType[] = [];
 
-const now = (): number =>
-  typeof performance !== 'undefined' ? performance.now() : Date.now() / 1000;
+const now = (): number => Date.now();
+
+const preciseNow = (): number =>
+  typeof performance !== 'undefined' ? performance.now() : Date.now();
 
 const add = (item: TimingItemType) => {
   log.push(item);
 };
 
 const start = (key: string) => {
   timingMap[key] = now();
+  preciseTimingMap[key] = preciseNow();
```

Comment:
If we had a precise timer, I would consider this suboptimal - we might have preferred a single map with objects as keys. Seeing we are not keeping the precision, I think we can not use that at all.